src/interact.ts

```diff
@@ -133,15 +135,17 @@ const interactViewPlugin = ViewPlugin.define<ViewState>((view) => ({
   // highlight a target (e.g. currently dragging or hovering)
   highlight(target) {
     if (target.rule.cursor) {
-      document.body.style.cursor = target.rule.cursor;
+      // Not ideal using !important but other cursor styles can be applied after.
+      // TODO: Find a way to apply this to the end of contentAttributes.
+      cursor.style = `cursor: ${target.rule.cursor} !important`;
```

Comment:
Does it work if you give the contentAttributes extension a `Prec.highest()`? I think `!important` is fine tho

Reply:
Also does this rely on codemirror keeping the reference of `cursor` object, I wonder if it's better to use a `Compartment` and reconfigure it in a transaction

Reply:
Updated to use a Compartment and `Prec.highest()`.  I think the Facet would support using the reference in that way  but I think this is much cleaner, good call.  Thanks